java/daikon/PptName.java

```diff
@@ -296,11 +296,37 @@ public int getPointSubscript() {
    * @return true iff this name refers to an abrupt completion point
    **/
   /*@EnsuresNonNullIf(result=true, expression="point")*/
+  /*@Pure*/ public boolean isThrowPoint() {
```

Comment:
Minor style point: the `@Pure` declaration annotation should go on its own line just as `@EnsuresNonNullIf` does.
(I realize it was already wrong in the code you started with.  It was corrected in commit 1409b8a4, which occurred after you started your work and which you haven't pulled into your branch yet.)